index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header>
+            <h2 class="logo">My Logo</h2>
+            <img class="hamburger" src="images/hamburger-menu.png" alt="hamburger menu" width=60>
+        </header>
+        
+        <section class="headline">
+            <h2>I am Jerry <br> Glad to see you!</h2>
```

Comment:
Please ensure that the main title ("I'm Jerry...") is an `h1` tag. The subtitle `(Glad to see you!)` can use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`